include/deal.II/grid/grid_tools.h

```diff
@@ -1285,6 +1296,93 @@ namespace GridTools
       const bool                                             perform_handshake,
       const bool enforce_unique_mapping = false);
 
+
+    /**
+     * Data structure returned by
+     * GridTools::internal::distributed_compute_intersection_locations(). It can
+     * be converted to
+     * GridTools::internal::DistributedComputePointLocationsInternal, which can
+     * be used to reinit Utilities::MPI::RemotePointEvaluation.
+     */
+    template <int dim, int spacedim>
```

Comment:
Is it `dim` or `structdim`. I would have said latter. It is the dimension of a geometric entity and not of a cell.

Reply:
I used ```dim``` as the dimension of the intersection and ```spacedim``` as the dimension of a geometric entity. I think this is consistent to ```compute_intersection_of_cells()``` https://github.com/dealii/dealii/blob/master/include/deal.II/cgal/intersections.h#L73

Reply:
As far as I can see `dim0` and `dim1` should be actually `structdim0` and `structdim1` (`structdim0<=dim0` and `structdim1<=dim1`).

Reply:
So I should use ```structdim``` here and change ```dim0``` and ```dim1``` from ```compute_intersection_of_cells()``` in another PR?

Reply:
Yes :+1:

Reply:
Resolved for this PR in [112676d](https://github.com/dealii/dealii/pull/15165/commits/112676d9cc46b7928d30682348e60e04375fd4ef) and added as a followup to https://github.com/dealii/dealii/issues/14338